timeline-chart/src/layer/time-graph-chart.ts

```diff
@@ -618,8 +626,27 @@ export class TimeGraphChart extends TimeGraphChartLayer {
             if (el instanceof TimeGraphStateComponent && !this.mousePanning && !this.mouseZooming) {
                 this.selectState(el.model);
             }
-            if (this.mouseInteractions && this.mouseInteractions.click) {
+            
+            var self = this;
+            this._mouseClicks++;
+
+            if (this._mouseClicks === 1) {
+                this._recentlyClickedGlobal = cloneDeep(e.data.global);
+                this._dblClickTimer = setTimeout(function(){
+                    self._mouseClicks = 0;
+                    self._recentlyClickedGlobal = null;
+                }, this._dbClickTime);
+            } else if (this._recentlyClickedGlobal && (this._recentlyClickedGlobal.equals(e.data.global))){
+                clearTimeout(this._dblClickTimer);
```

Comment:
I don't think we should clear on doubleClick, otherwise you get a doubleClick event on click(4), click(6), etc.

I think we should use a debouncer so every subsequent click on the same point (if close enough to previous click, not to first click...) will extend the timer. It's only when the debouce timer expires that we reset the click count and position. Or also if we get any click at a different position.

Minor point, if done like that then the dbClickTime constant should be renamed, as it's a timer for all multiple clicks (2,3,4,etc.)